### PR TITLE
Nominate Mariusz Sabath for 2024H1 SSC Election

### PR DESCRIPTION
**Pull request by faisal-memon**

### CODE REVIEW

File: ssc/elections/2024H1/MARIUSZ_SABATH.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# NOMINEE NAME HERE
```
Comment (amoore877) [inline ssc/elections/2024H1/MARIUSZ_SABATH.md:1]: name :)

File: ssc/elections/2024H1/MARIUSZ_SABATH.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+Mariusz has been a maintainer on the helm charts project since it started and has been a long time contributer to SPIRE.
+
+Be sure to preserve the two empty spaces at the end of each line below.  
+**GitHub Handle:** @mrsabath  
+**Email Address:** mrsabath@gmail.com  
+**LinkedIn Profile:** https://www.linkedin.com/in/mariusz-sabath-b36b0b20/  
+**Current Affiliation:** Unknown
```
Comment (amoore877) [inline ssc/elections/2024H1/MARIUSZ_SABATH.md:8]: no known affiliation? or actually unknown? we could ask him :) @mrsabath?

**Review by amoore877 — APPROVED** [review]
another PR can fix the relevant details :)

File: ssc/elections/2024H1/MARIUSZ_SABATH.md (hunk 2 above)
Comment (mrsabath) [reply ssc/elections/2024H1/MARIUSZ_SABATH.md:8]:
```suggestion
**Current Affiliation:** IBM Research 
```

**Pull request merged**